File: Frontend/src/server-models/cqrs/athletes/requests/update-athlete.request.ts
import { Gender } from 'src/server-models/enums/gender.enum';

export class UpdateAthleteRequest {
    id: string;
    username: string;
    email: string;
    firstname: string;
    lastname: string;
    fullName: string;
    gender: Gender;
    active: boolean;
}

File: Backend/Core/Backend.Application.Business/Business/Athletes/UpdateAthlete/UpdateAthleteRequest.cs
using System;
using Backend.Domain.Entities;
using Backend.Domain.Enum;
using MediatR;

namespace Backend.Application.Business.Business.Athletes.UpdateAthlete
{
    public class UpdateAthleteRequest : IRequest<Athlete>
    {
        public Guid Id { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
        public string FullName { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime LastModified => DateTime.UtcNow;
        public bool Active { get; set; }
    }
}
